client.go

```diff
@@ -16,6 +16,15 @@ func newClient(url string) (*StatusGoClient, error) {
 	return &StatusGoClient{rpcClient}, nil
 }
 
+func (c *StatusGoClient) nodeInfo() (*NodeInfo, error) {
```

Comment:
You can reduce it to:
```suggestion
func (c *StatusGoClient) nodeInfo() (info NodeInfo, err error) {
    err = c.rpcClient.Call(&info, "admin_nodeInfo")
    return
}
```

Reply:
Oh, nice! I like that.